indico/modules/rb_new/client/js/modules/landing/Landing.jsx

```diff
@@ -71,8 +88,12 @@ class Landing extends React.Component {
     };
 
     render() {
-        const {userHasFavorites} = this.props;
+        const {userHasFavorites, hasUpcomingBookings, upcomingBookings, fetchedUpcomingBookings} = this.props;
         const {extraState} = this.state;
+        const panes = () => [
```

Comment:
why a function?

Reply:
It's a trick to avoid rendering it unconditionally. Since I'm then rendering it within a ternary expression, it's easier than extracting the whole thing into its own function.

Reply:
Thought so, but I don't think just running the JSX snippets renders - so the ternary later should already avoid rendering it

Reply:
OK, I moved it all to a separate function, which makes things clearer.